invenio_communities/views/communities.py

```diff
@@ -185,6 +192,7 @@ def members(community=None, pid_value=None):
 
 @pass_community
 def invitations(community=None, pid_value=None):
+    endpoint = f"/api/requests/?created_by=community:{community['uuid']}"
```

Comment:
please check and apply similar https://github.com/inveniosoftware/invenio-communities/pull/486/files#diff-c8c30bf6f3d245d822fcd79d8a269b1d98295a779efa43050db4821a915ea625R29 to minimalize hardcoding URLs

Reply:
i am not sure if i implemented it correctly, so please check :smile: